.github/workflows/scheduled.yml

```diff
@@ -0,0 +1,77 @@
+name: scheduled
+
+on:
+  schedule:
+    # Run at 10:00 AM/PM UTC
+    - cron: "0 10,22 * * *"
+  workflow_dispatch:
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: run flake
+        uses: ./.github/actions/run-flake
+
+  isort:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: run isort
+        uses: ./.github/actions/run-isort
+
+  black:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: black
+        uses: psf/black@stable
+
+  common-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          - "3.11"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup
+        uses: ./.github/actions/setup
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: All tests
+        uses: ./.github/actions/all-tests
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

Comment:
@Alex-Izquierdo How do we get the coverage percentage displayed?

Reply:
Added badge